capstone/src/main/webapp/restaurantSingUp.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>STEP Capstone</title>
+    <script src="signUp.js"></script>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+  </head>
+  <body>
+    <h1>Sign Up as a Restaurant!</h1>
+    <div id="restaurant-sign-up">
+      <label for="new-restaurant-name">Name: </label>
+      <input type="text" id="new-restaurant-name" />
+      <br />
```

Comment:
nit: consider using styling here, rather than using line breaks to force layout.